Changes

```diff
@@ -61,6 +61,11 @@ Working version
 
 - Resurrect tabulation boxes in module Format. Rewrite/extend documentation
   of tabulation boxes.
+- PR#7515, GPR#1147: Arg.align now optionally uses the tab character '\t' to
```

Comment:
You should update the `review by...` part, break a line, and put this entry below the "update to map" one to respect the order (sorting by MPR first, GPR second).